yarn.lock

```diff
@@ -7768,6 +7786,19 @@ tuf-js@^1.1.3:
     "@tufjs/models" "1.0.3"
     make-fetch-happen "^11.0.1"
 
+twisted@^1.53.1:
+  version "1.53.1"
+  resolved "https://registry.yarnpkg.com/twisted/-/twisted-1.53.1.tgz#57593f9d01eb84271fc4c9cd020d2c4c913a9f4f"
+  integrity sha512-3uco5DiOGaagD+LN0o/LIwvc1+oO8EFZN3iMv7mMN5t4rV6xKPlkmfIp0K//yN4cmIBwffrAaaoJ6nGBEuVTfQ==
+  dependencies:
+    axios "^0.21.1"
```

Comment:
These people need to upgrade their dependencies. Maybe you can make a PR for that